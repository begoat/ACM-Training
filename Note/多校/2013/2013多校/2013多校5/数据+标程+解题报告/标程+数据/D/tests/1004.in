100
1526002
6904227
4303304
6194869
8256937
3952885
1254881
6414688
6364399
6596976
6097677
2448451
2699998
6537034
611379
3724330
8818938
3149106
7604676
2104825
9044721
3968276
5356795
5190002
5658075
1643348
9880091
7942379
3369850
8084367
2842677
9871386
611448
4673247
566241
7385802
5375277
1931286
2296961
4955685
9145303
2827775
4980227
491753
7847393
9183777
3379101
1003189
2061727
4537981
9659281
6080571
2193550
9923405
7915240
8537228
2308374
3416901
6764089
468434
1110857
5048532
2512992
7639950
6630947
759777
9529673
6355040
5747913
8589327
9973796
4779453
9134284
8293115
3958960
7637688
4923621
6382202
41330
6038715
3894072
5517278
2811004
9856076
6107155
3970224
6235817
3249099
1845293
5775348
1660252
3811632
7159657
3190890
2902854
3272655
5060370
1118408
3016479
9095957
