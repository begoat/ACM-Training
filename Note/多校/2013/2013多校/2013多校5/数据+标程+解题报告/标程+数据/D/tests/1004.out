0
0
0
511344
548288
322368
104572
0
964300
0
0
386592
0
0
0
0
0
0
0
159456
0
0
892800
0
0
0
1328384
1130976
0
0
0
0
0
0
0
0
0
0
183084
0
1457940
471296
661664
40978
652912
0
0
83600
249896
342616
803736
0
0
751760
0
0
0
0
563674
0
72864
0
0
0
884120
0
733048
0
0
0
0
0
0
1382186
0
0
0
0
0
0
0
0
0
0
1017860
0
415720
0
153772
0
0
0
477312
0
0
0
0
0
0
606392
